docs/optimization-framework/ab-testing.md

```diff
@@ -1,19 +1,14 @@
-# Experiments
+# A/B Tests
 
-Backed by [native Altis Analytics](./native.md), Experiments are a powerful tool for optimizing content and measuring the effectiveness of changes to the site.
+A/B Tests provides the tools to carry out randomized experiments using different variations of content features, eg: titles and featured images, and/or different variations of content blocks.
 
-The feature is enabled by default and provides a developer API for creating custom experiments as well as some built in features.
+Altis provides an A/B Test Block, which you can use to test different variations of a block / set of blocks. And also a developer friendly framework to test different content features like Title and Featured Images, which can typically be controlled from the Block Editor sidebar.
```

Comment:
Should never really start a sentence with "And", plus I think we can add a bit of detail e.g.:
```suggestion
Altis provides an A/B Test Block, which you can use to test different variations of a block or set of blocks in your content, widgets and layouts. There is also a developer friendly framework to add new custom tests that is used to power built in features like Title and Featured Image tests which can be controlled from the Block Editor sidebar.
```